RedditSharp/Reddit.cs

```diff
@@ -358,25 +371,25 @@ public Thing GetThingByFullname(string fullname)
             var data = WebAgent.GetResponseString(response.GetResponseStream());
             var json = JToken.Parse(data);
             return Thing.Parse(this, json["data"]["children"][0], WebAgent);
-        }
-
-        public Comment GetComment(string subreddit, string name, string linkName)
-        {
-            try
-            {
-                if (linkName.StartsWith("t3_"))
-                    linkName = linkName.Substring(3);
-                if (name.StartsWith("t1_"))
-                    name = name.Substring(3);
-
-                var url = string.Format(GetCommentUrl, subreddit, linkName, name);
-                return GetComment(new Uri(url));
-            }
-            catch (WebException)
-            {
-                return null;
-            }
-        }
+        }
+
+		public Comment GetComment(string subreddit, string name, string linkName)
+		{
+			try
+			{
+				if (linkName.StartsWith("t3_"))
+					linkName = linkName.Substring(3);
+				if (name.StartsWith("t1_"))
+					name = name.Substring(3);
+
+				var url = string.Format("http://reddit.com" + GetCommentUrl, subreddit, linkName, name);

```

Comment:
@SirCmpwn For some reason, the code errors for me without the explicit use of http://reddit.com because Uri() doesn't work without a fully qualified URL. Am I doing something wrong?